wallet/wallet.c

```diff
@@ -2684,7 +2699,8 @@ void wallet_filteredblock_add(struct wallet *w, const struct filteredblock *fb)
 	struct db_stmt *stmt;
 	if (wallet_have_block(w, fb->height))
 		return;
-	stmt = db_prepare_v2(w->db, SQL("INSERT OR IGNORE INTO blocks "
+
+	stmt = db_prepare_v2(w->db, SQL("INSERT INTO blocks "
 					"(height, hash, prev_hash) "
 					"VALUES (?, ?, ?);"));
```

Comment:
ON CONFLICT IGNORE?  Or was the "OR IGNORE" unnecessary?

Reply:
Turns out that given the above check for `wallet_have_block` and being in a DB transaction this should be unnecessary.